s3/sc-s3-synapse-ra.yaml

```diff
@@ -58,7 +59,11 @@ Resources:
             Sid: "ReadAccess"
             Effect: "Allow"
             Principal:
-              AWS: !Ref 'S3UserARNs'
+              AWS: !Split
+                - ","
+                - !Sub
+                  - "${list},arn:aws:iam::325565585839:root" 
```

Comment:
would it be possible to parameterize the synapse ARN and then ref it here like so..
```
- "${list},${SynapsAccountArn}"
```
That will allow us to easily pass in a different ARN in case it changes for some reason.

Reply:
So you want that to be a Parameter? I see your point, but that's another thing for the user to enter so opportunity for more user error. Could it be a Ref?

Reply:
SC will always show the param even when there's a default?  can it be hidden with adjustments to `AWS::CloudFormation::Interface` section?

Reply:
Agreed- I'll define a simple parameter at the top that can be overridden from the default if needed